src/site/content/en/react/codelab-quicklink/index.md

```diff
@@ -0,0 +1,192 @@
+
+The `manifest.json` file will be generated and made available at `https://site_url/rmanifest.json`.
+
+## Configure quicklink
+
+At this point you would need to install the Quicklink library in your project. For simplicity, we already added it to the project. Here's the command that you would need to run:
+
+```bash
+npm install --save quicklink
+```
+
+Open `src/components/App/index.js` for edit.
+
+First, import the Quicklink higher order component (HOC):
+
+```javascript/6/
+import React, { lazy, Suspense } from 'react';
+import { Route } from 'react-router-dom';
+
+import Footer from '@components/Footer';
+import Hero from '@components/Hero';
+import style from './index.module.css';
+import { withQuicklink } from 'quicklink/dist/react/hoc.js';
+
+const Home = lazy(() => import(/* webpackChunkName: "home" */ '@pages/Home'));
+const About = lazy(() => import(/* webpackChunkName: "about" */ '@pages/About'));
+const Article = lazy(() => import(/* webpackChunkName: "article" */ '@pages/Article'));
+const Blog = lazy(() => import(/* webpackChunkName: "blog" */ '@pages/Blog'));
+```
+
+Next, create an `options` object after the `Blog` variable declaration, to use as an argument when calling `quicklink`:
+
+```javascript/0-2/
+const options = {
+	origins: []
+};
+```
+Finally, wrap each route with the `withQuicklink()` higher order component, passing it an `options` parameter and the target component for that route:
+
+```javascript/5-11/
+const App = () => (
+  <div className={style.app}>
+    <Hero />
+    <main className={style.wrapper}>
+      <Suspense fallback={<div>Loading...</div>}>
+        <Route path="/" exact component={withQuicklink(Home, options)} />
+        <Route path="/blog" exact component={withQuicklink(Blog, options)} />
+        <Route
+          path="/blog/:title"
+          component={withQuicklink(Article, options)}
+        />
+        <Route path="/about" exact component={withQuicklink(About, options)} />
+      </Suspense>
+    </main>
+    <Footer />
+  </div>
+);
+```
+The previous code instructs to prefetch chunks for the routes wrapped with `withQuicklink()`, when the link comes into the view.
+
+{% Aside %}
+If Glitch throws an error at this point about the lack of a dependency,
+try opening the Glitch Terminal (**Tools** > **Terminal**), running
+`refresh` in the Terminal, then running `npm run build`.
+{% endAside %}
+
+## Measure again
+
+Repeat the first 6 steps from [Measure](#measure). Don't navigate to the blog page yet.
+
+When the home page loads the chunks for that route are loaded. After that, Quicklink prefetches the route's chunks for the in-viewport links:
+
+<img class="w-screenshot" src="./cdt-home-optimized.png" alt="Network panel showing the home page prefetching chunks.">
+
+These chunks are requested at the lowest priority and without blocking the page.
+
+Next: 
+
+1. Clear the Network log again.
+1. Click the **Blog** link to navigate to that page.
```

Comment:
```suggestion
1. Clear the Network log again.
1. Disable the **Disable cache** checkbox.
1. Click the **Blog** link to navigate to that page.
```

Reply:
@demianrenzulli FYI we need to enable the browser's caches again or else we won't see that the requests are coming from the prefetch cache